pages/common/redis-cli.md

```diff
@@ -20,4 +20,4 @@
 
 - Execute Redis command:
 
-`redis-cli {{redis command}}`
+`redis-cli {{redis_command}}`
```

Comment:
Nice catch. This is particularly troublesome in clients that make highlight tokens via color only (tldr-pages/tldr-node-client#18)